orchagent/main.cpp

```diff
@@ -13,42 +13,22 @@ extern "C" {
 #include <unistd.h>
 
 #include <sairedis.h>
+#include <logger.h>
+
 #include "orchdaemon.h"
-#include "logger.h"
+#include "saiapi.h"
 
 using namespace std;
 using namespace swss;
 
 extern sai_switch_notification_t switch_notifications;
 
-#define UNREFERENCED_PARAMETER(P)       (P)
+extern sai_switch_api_t *sai_switch_api;
+extern sai_router_interface_api_t *sai_router_intfs_api;
```

Comment:
i want to keep them here as i want to have all api externs in the current *.cpp file for easy track, similar to all the other *.cpp files. Later, I could remove these externs with a SaiApi class to have all the pointers.